back/src/test/java/com/baba/back/content/service/ContentServiceTest.java

```diff
@@ -732,4 +736,51 @@ class ContentServiceTest {
                 콘텐츠_제목_카드스타일_변경_요청_데이터))
                 .isInstanceOf(ContentBadRequestException.class);
     }
+
+    @Test
+    void 성장_앨범_댓글_삭제_시_댓글의_작성자가_아니면_예외를_던진다() {
+        // given
+        given(memberRepository.findById(멤버2.getId())).willReturn(Optional.of(멤버2));
+        given(contentRepository.findById(컨텐츠10.getId())).willReturn(Optional.of(컨텐츠10));
+        given(babyRepository.findById(아기1.getId())).willReturn(Optional.of(아기1));
+        given(relationRepository.findByMemberAndBaby(멤버2, 아기1)).willReturn(Optional.of(관계11));
+        given(commentRepository.findById(댓글10.getId())).willReturn(Optional.of(댓글10));
+
+        // when & then
+        assertThatThrownBy(() -> contentService.deleteComment(멤버2.getId(), 아기1.getId(), 컨텐츠10.getId(), 댓글10.getId()))
+                .isInstanceOf(CommentBadRequestException.class);
+    }
+
+    @Test
+    void 성장_앨범_댓글을_삭제할_수_있다() {
+        // given
+        given(memberRepository.findById(멤버1.getId())).willReturn(Optional.of(멤버1));
+        given(contentRepository.findById(컨텐츠10.getId())).willReturn(Optional.of(컨텐츠10));
+        given(babyRepository.findById(아기1.getId())).willReturn(Optional.of(아기1));
+        given(relationRepository.findByMemberAndBaby(멤버1, 아기1)).willReturn(Optional.of(관계11));
+        given(commentRepository.findById(댓글10.getId())).willReturn(Optional.of(댓글10));
+
+        // when & then
+        assertAll(
+                () -> assertThatCode(
+                        () -> contentService.deleteComment(멤버1.getId(), 아기1.getId(), 컨텐츠10.getId(), 댓글10.getId()))
+                        .doesNotThrowAnyException(),
+                () -> verify(commentRepository).delete(댓글10),
+                () -> verify(tagRepository).deleteByComment(댓글10)
```

Comment:
```suggestion
                () -> then(commentRepository).should().delete(댓글10),
                () -> then(tagRepository).should().deleteByComment(댓글10)
```

BDDMockito 스타일로 통일하는게 좋을 것 같아요!

Reply:
네 변경햇습니다~